crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -53,7 +53,7 @@ sol! {
         }
 
         #[derive(Default)]
-        struct FuzzAbiSelector {
+        struct FuzzArtifactSelector {
             string contract_abi;
```

Comment:
can we rename the field too

Reply:
Oh, right, done in https://github.com/foundry-rs/foundry/pull/7850/commits/d2da2f64726c313728abb602999c1de9af41eb81